commons/passive/dreem/dreem_sleependpoints.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "Dreem",
+  "doc": "AVRO Schema for DREEM sleep endpoints",
```

Comment:
What are DREEM endpoints, concretely? Is it a feature? If needed place a link. In any case, endpoint is not a well-known phrase for a feature, statistic or aggregation.